### ISSUE TITLE
Green screen instead of the live preview

### ISSUE THREAD

**Issue by xmrhahnx** [mebjas/html5-qrcode#573]
**Describe the bug**
Green screen showed instead of the live camera stream.

**Expected behavior**
See the live preview from the camera.

**Screenshots**
[Screenshot](https://i.imgur.com/8YjTTPr.jpg)

**Smartphone:**
 - Device: Samsung S8 (SM-G950F)
 - OS: Android 9
 - Browser: Chrome 
 - Version: 107.0.5304.91

**Additional context**
The **QR code is correctly scanned anyway** but there is no live preview.

**Comment by ROBERT-MCDOWELL** [comment]
maybe an issue with the resolution of your camera?

**Comment by xmrhahnx** [comment]
I don't think so. I tested this library on a Samsung Galaxy J3 2017 (Android  9 and Chrome 106.0.5249.126) and it works.

**Comment by ROBERT-MCDOWELL** [comment]
I mean samsung browser that cannot get the default camera resolution

**Comment by xmrhahnx** [comment]
I tried with the Samsung proprietary web browser and it works. So, it seems that there is some compatibility issues with that version of Chrome. Is anyone facing the same issue?

**Comment by xmrhahnx** [comment]
Maybe it is a Chrome bug with navigator.mediaDevice.getUserMedia method itself. 
Even if I just try to create a video object, assigning stream object as its source, it shows just a green static screen.
Code below.

HTML:
`<video autoplay="true" width="100%" id="video-test"></video>`

JS:
```
var video = document.querySelector("#video-test");

navigator.mediaDevices.getUserMedia({
      video: {
        mandatory: { minFrameRate: 10, minWidth: 100, minHeigth: 100  },
      }
 })
 .then(function (stream) {
      video.srcObject = stream;
 })
 .catch(function (err0r) {
      console.log("Something went wrong!", err0r);
 });
```

**Comment by ROBERT-MCDOWELL** [comment]
so I suggest to close this issue and open an issue at chrome issue tracker.
